src/main/java/com/ldtteam/structurize/placement/BlueprintIterator.java

```diff
@@ -47,6 +48,19 @@
      */
     private boolean isRemoving;
 
+    /**
+     * Iteration version (1 = original, 2 = outside in).
+     */
+    private static int ITERATION_VERSION = Structurize.getConfig().getServer().iterationVersion.get();
```

Comment:
This does not work!

Server config is not available on the load of this class as such the static value will most likely differ between client and server!

Reply:
This only runs on the server.

Reply:
Or do you mean singleplayer/smp?

Reply:
I think the point is that in theory, static constructors run at mod init, which is before the config files are read.  (In practice, static constructors run whenever the class is actually loaded, which is more fragile / less predictable -- and for this particular class, probably does work by happenstance.)

Reply:
@uecasm Is correct.
Additionally this completely breaks in SMP/SIngleplayer.

With my PR to forge you can just query it all the time, there is virtually no performance benefit to doing it this way.

Reply:
I don't do this for performance, I don't want it changed during runtime.

Reply:
Then don't make it configurable.

The forge config system was designed around mutability of these values.
Especially the Server configuration has this in mind since it is synchronized from the client.

What is the exact reason you don't want this to change during runtime? I mean I can see why you don't want it to change during an iteration/buildprocess. But when no build is going on, does it matter that the value changes?